tfx/proto/example_gen.proto

```diff
@@ -46,6 +46,11 @@ message Input {
   //   - This spec matches digits and maps the data into the relevant SPAN
   //     numbers. e.g. 'data_{SPAN}-*.tfrecord' will collect files like
   //     'data_12-a.tfrecord', 'date_12-b.tfrecord'.
+  //   - Optionally, it could be specified with the width of the integers
+  //     when mapped.
+  //     e.g. 'data_{SPAN:2}.file' mas to files like 'data_02.file' and
+  //       'data_27.file' (as inputs for Apan-2 and Span-2 respectively),
```

Comment:
nit, s/Apan-2/Span-2 and s/Span-2/Span-27

Reply:
fixed